test/aws/awscore_test.yml

```diff
@@ -93,3 +93,42 @@ Resources:
               - sqs:SetQueueAttributes
             Resource:
               - !Sub arn:aws:sqs:*:${AWS::AccountId}:ocaws-jl-test-queue-*
+
+  # Policies to support testing on EC2
+  EC2StackPolicy:
+    Type: AWS::IAM::Policy
+    Properties:
+      PolicyName: EC2StackPolicy
+      Users:
+        - !Sub ${PublicCIUser}
+      PolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
```

Comment:
This needs to be locked down significantly. The user should only be allowed to perform cheap actions. 

What we probably want to do is create the security group and roles in advance in this template, and create a Launch Template that the PublicCIUser has permissions to run.

Reply:
The launch template can also be created in this template, but here's an interactive guide for an overview of what they are and now they work: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-launch-templates.html